### ISSUE TITLE
Problem selecting an embayment and then switching habitat types

### ISSUE THREAD

**Issue by walkerjeffd** [HWR-llc/ett#75]
Some odd behavior when I have an embayment selected, and then I change habitat type.

Starting with Eelgrass, I select Annisquam.

<img width="1430" alt="image" src="https://user-images.githubusercontent.com/981144/171776901-2911856c-e679-4273-8f66-356432834719.png">

Then I click on Salt Marsh radio button, and Annisquam is still shown on bar chart, but the polygon no longer has red outline on the map. Bar chart shows 639 and 636 acres.

<img width="1436" alt="image" src="https://user-images.githubusercontent.com/981144/171777090-67336f63-65ad-438a-b2b9-5a9345ce8e12.png">

Then I click on the Annisquam polygon (trying to select it again), but I'm actually unselecting it so there is still no red outline. Bar chart now shows 33,767 and 33,235 acres for "All Assessment Areas". So looks like I have in fact unselected it, but it wasn't clear that it was already selected because it did not have a red outline after I switched from Eelgrass to Salt Marsh.

<img width="1413" alt="image" src="https://user-images.githubusercontent.com/981144/171777458-c7cc0947-6e98-4d3b-9666-c3028f8742cf.png">

Bottom line is that if an embayment is selected for a given habitat type, and the user then changes the habitat type, the data for that embayment is still shown on the bar chart, but it is no longer shown as selected on the map (no red outline).
